src/pspm_options.m

```diff
@@ -271,7 +271,7 @@
     options = autofill(options, 'norm',                   0,          1                 );
     options = autofill(options, 'overwrite',              0,          1                 );
     options = autofill(options, 'marker_chan_num',        'marker',   '*Num*Char'       );
-    options = autofill(options, 'marker_chan_num_event',  'first',    '*Num*Char'       );
+    options = autofill(options, 'marker_chan_num_event',  'last',     '*Num*Char'       );
```

Comment:
I am a little confused about the suggestion.
I agree that this line can be removed if the user is no longer using `last` or `first` to specify marker channels.
However, you also say "Either the user specifies the marker channel number as a numeric (i.e. the n-th channel out of all channels) or the first marker channel is used." (https://github.com/bachlab/PsPM/pull/493/files#r1237991430). Without specifying `marker_chan_num_event`, I am unsure how I am expected to implement this?
You also suggest to keep "This should be put back in, but changed to data{1}, because we always want to use the first marker channel." at line 264 (https://github.com/bachlab/PsPM/pull/493/files#r1237991145). This seems to mean always the first marker channel should be chosen. I don't quite understand if you would still suggest to allow "user specifies the marker channel number as a numeric", if my understanding of the two sentences are correct?